src/Symfony/Component/Security/Core/Authorization/Voter/TraceableVoter.php

```diff
@@ -39,7 +39,7 @@ public function __construct(VoterInterface $voter, EventDispatcherInterface $eve
         }
     }
 
-    public function vote(TokenInterface $token, $subject, array $attributes): int
+    public function vote(TokenInterface $token, $subject, array $attributes)
```

Comment:
can you add an `@return` annotation please?
Note that in Symfony 6, returning an `int` will be enforced, so that you should also fix any code that doesn't return an `int` already.

Reply:
Is it needed, given that this is already part of `VoterInterface::vote()`?

Reply:
@nicolas-grekas as mention above. It is already in interface.

Note taken. Actually, it is already well discussed in related issue.

Reply:
OK, indeed.